osv/malicious/pypi/veilcord-tls/MAL-0000-veilcord-tls.json

```diff
@@ -0,0 +1,42 @@
+{
+  "modified": "2025-09-17T04:05:07Z",
+  "published": "2025-09-17T04:05:07Z",
+  "schema_version": "1.5.0",
+  "id": "",
+  "summary": "Malicious code in veilcord-tls (PyPI)",
+  "details": "This package decodes a payload and executes it whenever it is imported. It seems to be targeting veilcord package users. Its contents are almost identical to veilcord, except for the addition of the malicious payload.\n",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "PyPI",
+        "name": "veilcord-tls",
+        "purl": "pkg:pypi/veilcord-tls"
+      },
+      "versions": [
+        "0.0.7.5"
```

Comment:
"0.0.7.6" also appears to be malicious too.

Reply:
Thanks for spotting this. I have added the missed version.